Makefile

```diff
@@ -42,6 +42,13 @@ clean-mocks:
 	rm -rf mocks
 .PHONY: clean-mocks
 
+EVM_EVENTS_ORIGIN:="docker/deployed/testnet/api/backup_database.db"
+EVM_EVENTS_TARGET:="pkg/eventprocessor/impl/testdata/evm_history.db"
+generate-history-db:
+	rm -f ${EVM_EVENTS_TARGET}
+	sqlite3 ${EVM_EVENTS_ORIGIN} 'ATTACH DATABASE ${EVM_EVENTS_TARGET} as target' 'CREATE TABLE target.system_evm_events as select * from system_evm_events'
+	zstd -f ${EVM_EVENTS_TARGET}
+	rm ${EVM_EVENTS_TARGET}
```

Comment:
Nope, it isn't retrieved by `scp`. In the cloud validator we don't have a full history of events, since that validator started persisted events when the feature came into existence and forward, so it'd have events missing.

The database I used was from my local validator, which I synced from scratch after the PR saving events was created, so AFAIK is the only one that has all the events.

Whenever we do a cold sync of the cloud validator, yes, we should retrieve it with `scp` and use that.

Reg the datetime, we could but technically speaking the `block_number` of the events already give a timestamp of the events in the chains. Or maybe you were thinking about date-times for other reason?

Reply:
The datetime was mostly for us to visually know the date of the backup. But I was assuming it came from a remote backup. But makes sense it was from your local machine.